test/validate/ferc1_test.py

```diff
@@ -20,7 +20,7 @@
     "plant_in_service_ferc1",
 ]
 unique_record_tables = [
-    t for t in PUDL_TABLES["ferc1"] if t not in row_mapped_tables
+    t for t in Ferc1Settings.tables if t not in row_mapped_tables
```

Comment:
Should this be referring to the list of all possible working tables, rather than just the ones that happen to be contained in this `Ferc1Settings` object?

Reply:
Yeah I think so. I'll change this